# DP: Description: Allow ldconfig to work on i386/x86-64 biarch systems
# DP: Related bugs: 
# DP: Dpatch author: Daniel Jacobowitz
# DP: Patch author: Daniel Jacobowitz
# DP: Upstream status: Debian-Specific
# DP: Status Details: based on Ubuntu change by Jeff Bailey
# DP: Date: 2005-10-13

--- /dev/null	2005-07-29 18:11:21.000000000 +0000
+++ glibc-2.3.5/sysdeps/unix/sysv/linux/i386/dl-cache.h	2005-08-05 14:15:39.162794976 +0000
@@ -0,0 +1,1 @@
+#include <sysdeps/unix/sysv/linux/sparc/dl-cache.h>
